app/presenters/content_item_with_parts_presenter.rb

```diff
@@ -0,0 +1,83 @@
+class ContentItemWithPartsPresenter < ContentItemPresenter
```

Comment:
For the other formats we opted to name them after the specific format (ie AnswerPresenter, LocalTransactionPresenter etc). We thought that would be simpler to separate the formats in future (should anyone want to pull out any format into another app). So we could rename this accordingly. Also the other parted format Programmes will go away, so we won't benefit from reusing this anyway.